docker-compose.web.yml

```diff
@@ -0,0 +1,44 @@
+version: '3'
+services:
+  admin-app:
+    build: ./admin
+    container_name: mtc_admin
```

Comment:
With `container_name`, the services in this compose file will conflict with others in `docker-compose.admin-test.yml` and `docker-compose.pupil-test.yml` (that’s why the ones in pupil-test have _2). Same on lines 16 and 31 also.

Reply:
after trying to work with these files i'm of the opinion that we need to work out a better way to manage the separate test runs.  Lots of duplication etc.  I'm open to ideas.